bitmap-destruction-force/lib/bitmapFileHelper.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const fs = require('fs');
+var buf;
+module.exports = exports = {};
+
+exports.turnGreen = function(callback) {
+  fs.readFile(`${__dirname}/../../assets/bitmap.bmp`, function(err, data){
+    if(err) throw new Error('bitmap buffer not acquired');
```

Comment:
Instead of throwing errors, you should pass them to a callback and deal with them in that way.